pkg/proc/target.go

```diff
@@ -78,6 +78,8 @@ type Target struct {
 	fakeMemoryRegistry    []*compositeMemory
 	fakeMemoryRegistryMap map[string]*compositeMemory
 
+	moduleDataCache []ModuleData
```

Comment:
Maybe this type would make more sense on `BinaryInfo` directly so we don't have to add the extra param below.

Reply:
Done.